### DISCUSSION TITLE
How to set index page to 'about' using github pages

### DISCUSSION THREAD

**Discussion by chrisjameslennon** [mmistakes/minimal-mistakes#3144] [Q&A] [answered]
Hi there

Sorry for the question here, but I have done a lot of googling and experimenting on this one, I just cannot get this to work

How would I make the index (home) page the 'about' page (or any other page) when using github pages.

I have tried updating the theme file that is referenced in index.html to include the about page.  I think the fact that the about page is an md file means it is rendering just as text, not rendered html.  See https://chrisjameslennon.github.io/mm/

I would like to do these purely in github, i.e. not have do do any build stuff, i.e. just by changing the files in github

Is this possible and how to do this?

Any help or advice really appreciated

Chris

**Comment by mmistakes** [comment, marked as answer]
The quickest way is take your about page Markdown file... I'm going to assume it's named something like `about.md`.
In that file, withing the YAML front matter (the stuff that begins with `---`) add/or change the permalink to `permalink: /` or `permalink: index.html`.

This will then tell Jekyll to output your About page as `/index.html` which is your site's "home page".
After you do this you'll need to remove `index.md` or `index.html` in the root of your repo. Or if you have a `_pages` folder anything that might be named `index.md` or `home.md`. If you leave any of these pages they will likely overwrite the About page index file and you'll still have a page with your recent posts.

**Reply by chrisjameslennon** [reply]
Thank you, that worked.  I don't think I would have figured that out myself!

**Comment by mmistakes** [comment]
If you want more detail on how `permalink` works check this post I wrote recently.
https://mademistakes.com/mastering-jekyll/how-to-link/
